### PR TITLE
[ADD] web_dashboard_open_action

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

**Comment by hbrunn** [comment]
`oe_maximize` is already taken by the inverse of the minimize icon (the plus you get when you press the middle button from you screenshot)

I looked through the fonts the odoo provides, but couldn't find anything usable. Fontawesome used to have a good icon for that: http://fortawesome.github.io/Font-Awesome/3.2.1/icon/fullscreen, but I can't seem to find it in odoo's version of it

**Comment by pedrobaeza** [comment]
`fa-icon-fullscreen` doesn't serve?

**Comment by OSguard** [comment]
I test the module. Their is a functional problem.
- chosse some report
- apply a filter
- add to dashboard
- in dashboard the filter is used
- click on new "open_action"
- Expected Result: Filter is still used
- Result now: Filter is ignored

**Comment by hbrunn** [comment]
@OSguard thanks, that's fixed now
@pedrobaeza yes, good idea, what confuxed me is that it's http://fortawesome.github.io/Font-Awesome/icon/arrows-alt in odoo's copy of fontawesome

**Comment by StefanRijnhart** [comment]
@hbrunn I was expecting to see a new commit related to @osguard's observation. Do I miss something?

**Comment by hbrunn** [comment]
@StefanRijnhart and you're perfectly right about that. Now the missing commits are pushed to the right place

File: web_dashboard_open_action/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "Open a dashboard's action",
+    "version": "1.0",
+    "author": "Therp BV",
```
Comment (OSguard) [inline web_dashboard_open_action/__openerp__.py]: add OCA as Author

**Comment by OSguard** [comment]
besides the missing author :+1: (test & review)

**Comment by lmignon** [comment]
:+1:  (test & review)

File: web_dashboard_open_action/README.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+=========================
+
+This module adds a button to open a dashboard's action in the usual view, thus enabling filtering and any other actions you can apply on the resulting records. Basically, this is the inverso to `Add to dashboard`.
+
+Usage
+=====
+
+After installation, a dashboard has a `Maximize`-button that loads the dashboard's action in a normal view.
+
+Credits
```
Comment (StefanRijnhart) [inline web_dashboard_open_action/README.rst:20]: The OCA template has now been extended with a Bug Tracker section containing a template url in which you are supposed to substitute the repo and module names:

https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

**Pull request merged**